src/Plugin/Validation/Constraint/UniqueOgMembershipConstraint.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\og\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Entity Reference valid reference constraint.
+ *
+ * Verifies that referenced entities are valid.
+ *
+ * @Constraint(
+ *   id = "UniqueOgMembership",
+ *   label = @Translation("Unique OG membership", context = "Validation")
+ * )
+ */
+class UniqueOgMembershipConstraint extends Constraint {
+
+  /**
+   * Not a unique membership group message.
+   *
+   * @var string
+   */
+  public $NotUniqueMembership = 'The user %user is already a member in this group';
```

Comment:
Class property $NotUniqueMembership should use lowerCamel naming without underscores:
```suggestion
  public $notUniqueMembership = 'The user %user is already a member in this group';
```